OLD-README.md

```diff
@@ -83,18 +83,18 @@ This is a guide on how to configure your project manually with a single file `co
 		*  [output](#template-output)
 
 ## Commit0.yaml<a name="commit0-yaml"></a>
-Your project config file. It describes the project 
+Your project config file. It describes the project
 Example:
 ```
 name: newProject
-context: 
+context:
   cognitoPoolID: xxx
-modules: 
-  - source: "github.com/commitdev/commit0-aws-eks-stack"	
+modules:
+  - source: "github.com/commitdev/commit0-aws-eks-stack"
     output: "infrastructure"
-  - source: "github.com/zthomas/react-mui-kit"	
+  - source: "github.com/zthomas/react-mui-kit"
```

Comment:
Did we want to replace this with the other Frontend stack?

Reply:
That's the old readme that can probably be deleted but I figured we should keep around for a little bit for reference.